src/Propel/Runtime/Formatter/ArrayFormatter.php

```diff
@@ -46,7 +59,7 @@ public function format(DataFetcherInterface $dataFetcher = null)
         foreach ($dataFetcher as $row) {
             $object = &$this->getStructuredArrayFromRow($row);
             if ($object) {
-                $items[] =& $object;
+                $items[] = & $object;
```

Comment:
isn't it should be together?

Reply:
From techn. perspective this doesnt matter. But code style wise I think it should belong to the var even (without space).
A follow up CS fixer update IMO.